pkgs/tools/security/bitwarden/default.nix

```diff
@@ -102,6 +112,13 @@ buildNpmPackage' {
     python3
   ];
 
+  postPatch = ''
+    if [[ $(jq --raw-output '.devDependencies.electron' < package.json | grep -E --only-matching '^[0-9]+') != ${lib.escapeShellArg (lib.versions.major electron.version)} ]]; then
+      echo 'ERROR: electron version mismatch'
+      exit 1
+    fi
+  '';
+
   preBuild = ''
```

Comment:
I think we want to move that to a later step in the build to not break the vendoring

```
patching sources
/nix/store/f77w5n01j0s8qlhniw79ms0pv1dpyh80-stdenv-linux/setup: line 119: jq: command not found
ERROR: electron version mismatch
configuring
error: builder for '/nix/store/i9cxlz7isqj78bzbva74a9a4fikf36q4-bitwarden-2023.4.0-npm-deps.drv' failed with exit code 1;
       last 10 log lines:
       > Hunk #7 succeeded at 39152 (offset 1 line).
       > Hunk #8 succeeded at 71235 (offset 2 lines).
       > Hunk #9 succeeded at 71251 (offset 2 lines).
       > Hunk #10 succeeded at 71274 (offset 2 lines).
       > Hunk #11 succeeded at 71664 (offset 2 lines).
       > Hunk #12 succeeded at 74106 (offset 2 lines).
       > patching file package.json
       > patching file package.json
       > /nix/store/f77w5n01j0s8qlhniw79ms0pv1dpyh80-stdenv-linux/setup: line 119: jq: command not found
       > ERROR: electron version mismatch
       For full logs, run 'nix log /nix/store/i9cxlz7isqj78bzbva74a9a4fikf36q4-bitwarden-2023.4.0-npm-deps.drv'.
no configure script, doing nothing
building
ERROR: The Cargo.lock file doesn't exist
Cargo.lock is needed to make sure that cargoHash/cargoSha256 doesn't change
when the registry is updated.
```